modules/ossm-install-cert-manager.adoc

```diff
@@ -0,0 +1,161 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-install-cert-manager_{context}"]
+== Install cert-manager
+
+To install cert-manager, follow these steps:
```

Comment:
Please update the steps below based on my new PR (look at istio-csr directory): https://github.com/maistra/istio-operator/pull/1156. These samples were tested with cert-manager operator and istio-csr v0.6.0.